draft-ietf-oauth-transaction-tokens.md

```diff
@@ -663,6 +669,14 @@ The following entry will be proposed using the IANA Media Type registration {{IA
 * Intended Usage: Common
 * Contact Person: Atul Tulshibagwale
 
+## HTTP Header
+The header name `Txn-Token` is proposed to be added to the HTTP Field Name Registry {{IANA.HTTP.FieldNames}}. This header is defined in the section {{txn-token-http-header}}. The following entry will be proposed in the HTTP Field Name Registry:
+
+* Field Name: Txn-Token
+* Status: permanent
+* Specification Document: Section {{txn-token-http-header}} of this document
+* Comment: The `Authorization` header cannot be used for Txn-tokens because that may be use for service-to-service authorization, and the services may simulatneously require the use of Txn-tokens to convey detailed immutable information such as user identity and details of fine-grained authorization that are included in the Txn-token.
```

Comment:
> typo ... use -> used 

that and one other spelling thing

```suggestion
* Comment: The `Authorization` header cannot be used for Txn-tokens because that may be used for service-to-service authorization, and the services may simultaneously require the use of Txn-tokens to convey detailed immutable information such as user identity and details of fine-grained authorization that are included in the Txn-token.
```